.github/workflows/other.yml

```
@@ -0,0 +1,66 @@
+
+      - name: Show build number
+        run: |
+          echo "Build Number: ${{ github.run_number }}"
+      - name: Show pull request number
+        if: ${{ github.event_name == 'pull_request' }}
+        run: |
+          echo "PR Number: ${{ github.event.pull_request.number }}"
+      - name: Set Docker Image Tag
+        id: set_docker_image_tag
+        env:
+          PR_NUMBER: ${{ steps.get_pr_number.outputs.PR_NUMBER }}
```

actionlint (1.7.4)

property "get_pr_number" is not defined in object type {}

(expression)